lib/tasks/add_browse_users.rake

```diff
@@ -0,0 +1,42 @@
+# rake organisation:add_browse_users
+namespace :organisation do
+  task add_browse_users: :environment do
```

Comment:
please add a description block here so that it shows up when you type 'rake -T'.